internal/cloud/mutagenbox/forward.go

```diff
@@ -13,7 +13,7 @@ import (
 
 func ForwardCreate(host, localPort, remotePort string) (string, error) {
 	var err error
-	if localPort == "" {
+	if localPort == "" || localPort == "0" {
 		localPort, err = getFreePort()
```

Comment:
`0` usually already means "find an open port" in linux networking, so you might be able to get rid of `getFreePort` and just pass `0` along. Worth trying.

Reply:
will try

Reply:
nope, mutagen just uses 0 and fails silently :(